prod/relayers.yml

```
@@ -0,0 +1,91 @@
+---
+- name: Build relay binary locally
+  hosts: 127.0.0.1
+  connection: local
+
+  tasks:
+    - name: Build binary
+      ansible.builtin.shell:
+        cmd: GOOS=linux GOARCH=arm64 mage build
+        chdir: ../../snowbridge/relayer
+
+- name: Update relayers
+  hosts: relayers
+  remote_user: root
+  become: true
+  become_method: sudo
+
+  vars:
+    relays:
+      - name: beefy-relay
+        enabled: false
+        state: stopped
+      - name: beacon-relay
+        enabled: false
+        state: stopped
+      - name: asset-hub-parachain-relay
+        enabled: false
+        state: stopped
+      - name: asset-hub-ethereum-relay
+        enabled: false
+        state: stopped
+    endpoints:
+      beacon: https://ip-172-31-42-147.eu-central-1.compute.internal:9596
```

@vgeddes, initially this should be:

```suggestion
      beacon: https://lodestar-mainnet.chainsafe.io/
```

Once the relayer has caught up to 9 May, we need to switch this to `https://ip-172-31-42-147.eu-central-1.compute.internal:9596`. I know you want to prevent manual fiddling, just want to give you a heads-up.

Once the beacon relayer has caught up until 9 May, it will start failing since it won't find the required ssz states in the beacon store.